lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1006,7 +1009,7 @@ def view_init(self, elev=None, azim=None, roll=None, vertical_axis="z"):
             The axis to align vertically. *azim* rotates about this axis.
         """
 
-        self.dist = 10
+        self._dist = 10  # The camera distance from origin. Behaves like zoom
```

Comment:
Is it ok to make `dist` private? The user doesn't interact with it anywhere

Reply:
Unfortunately not. It's formally public and since we have an extremely large user base, chances are that somebody uses that. We have been bitten by such changes often enough that we're quite strict on API changes.

What you can do is deprecate the public `self.dist`. To do so add `dist = deprecate_privatize_attribute("3.6")` to the class and add a deprecation note in `doc/api/next_api_changes/deprecations`

Reply:
That's some pretty nifty deprecation handling where it automatically renames the attribute. I think it makes sense to privatize that. Good for review now.